.travis.yml

```diff
@@ -15,6 +15,6 @@ install:
   - rm qiime2-2018.6-py35-linux-conda.yml
```

Comment:
after `rm qiime2-2018.6-py35-linux-conda.yml` in a new line add `source activate qiime2-2018.6`

Reply:
done

Reply:
But not sure why the travis check still fails.